src/robusta/core/sinks/robusta/dal/model_conversion.py

```diff
@@ -97,7 +97,7 @@ def to_evidence_json(
                 structured_data.append({"type": "list", "data": block.items})
             elif isinstance(block, PrometheusBlock):
                 structured_data.append(
-                    {"type": "prometheus", "data": block.data.dict(), "metadata": block.metadata, "version": 1.0}
+                    {"type": "prometheus", "data": block.data.json(), "metadata": block.metadata, "version": 1.0}
```

Comment:
@Avi-Robusta is that on purpose? 
@RoiGlinik fixed it few days ago (turning it from json to dict)

Reply:
The UI graphs stopped working after the change in staging. I'll check again to see if it has been supported in the past two days